var/spack/repos/builtin/packages/python/package.py

```diff
@@ -642,6 +643,16 @@ def command(self):
             path = os.path.join(self.prefix.bin, 'python{0}'.format(ver))
             if os.path.exists(path):
                 return Executable(path)
+            else:
+                path = self.prefix.bin.replace(os.sep+"bin","")
```

Comment:
I don't really understand this logic. To me it looks like this is equivalent to `self.prefix` (without the `.bin`). You don't even use this variable anyway, so you can delete it.

Reply:
Thanks for the review. This is left over from an earlier draft and should not be there.